### ISSUE TITLE
Turn off Genesis Mini menu music

### ISSUE THREAD

**Issue by niZ31** [TeamShinkansen/Hakchi2-CE#377]
I just want to simply turn it off. Tried the Advanced Music Hack with a silent WAV file(16-bit PCM) in both folders and it just doesn't work, menu music still plays normally. Please make this a feature.

**Comment by padshance** [comment]
i agree, we can listen the song once, it seems cool, twice or three times... nop, i also did the same but with some classic songs, mp3, wav etc etc, nothing works... thanks in advance

**Comment by KMFDManic** [comment]
@niZ31 @padshance Unfortunately, Advanced Music Hack ONLY works with SNES/NES.  We don't currently have true ability to manipulate the assets of the main UI of MD Mini.  It actually would have been a good idea for them to have it as a "true" menu option to disable it, for those who don't want it.  But, as of right now, certain things cannot be done on MD, for the time being.  This may or may not change, in the future.  But, we are already aware of requests like this.  It is just a limitation of what can actually be done with the hardware and custom kernel, for now.

**Comment by DanTheMan827** [comment]
@madmonkey1907
